h/__main__.py

```diff
@@ -1,4 +1,4 @@
-from h.cli import main
+from h.cli import main  # pragma: no cover
```

Comment:
I'm going to sprinkle a few of this to make the missing coverage report shorter.